src/ages/age-one/index.ts

```diff
@@ -1,20 +1,21 @@
+/* eslint-disable no-console */
+
 import { formatUnits } from "ethers/lib/utils";
-import { fetchUsers } from "../../graph/fetch";
+import { fetchUsers, userBalancesToUnclaimedTokens, computeMerkleTree, getAccumulatedEmission } from "../../utils";
 import * as fs from "fs";
 import path from "path";
-import { computeMerkleTree } from "../../computations/compute-merkle-tree";
-import configuration from "./configuration";
-import { getMarketsEmission } from "./getMarketsEmission";
-import { userBalancesToUnclaimedTokens } from "./getUserUnclaimedTokens";
+import { ageOneDistribution } from "../distributions";
 import { BigNumber } from "ethers";
-import { now } from "../../helpers/time";
+import { now } from "../../helpers";
+import { ages } from "../ages";
+
 const main = async (ageName: string, _epoch: string) => {
+  const configuration = ages[0];
   console.log("Compute markets parameters");
-  if (!Object.keys(configuration.epochs).includes(_epoch)) throw Error("invalid epoch name");
-  const epoch = _epoch as keyof typeof configuration.epochs;
-  const epochConfig = configuration.epochs[epoch];
+  const epochConfig = configuration.epochs.find((e) => e.epochName === _epoch);
```

Comment:
we could expect an an ageIndex and an epochIndex as input here?

Reply:
mmmh this script is now outdated and will be refactored in a future PR soon